packages/config/tailwind/index.ts

```diff
@@ -9,6 +11,207 @@ export const tailwindConfig: Config = {
   },
   theme: {
     extend: {
+      typography: ({ theme }: ThemeConfig) => ({
```

Comment:
question: I thought that OUR prose styles should be inside our plugin, so users would be able just to install our plugin and have our prose styling, I'm confused here a bit :)

Reply:
Not sure if I understand correctly, do you mean - as part of SfTypography created by @FRSgit ?

Reply:
I think that one was made clear during the standup - is there anything else we can do to push this PR forward?

Reply:
@FRSgit everything is clear and thank you and @aniamusial for explanation :)